File: Lista 5/L5_Q6.c
#include <stdio.h>

int main(){
	
	float premio, g1, g2, g3;
	premio = 780000;
	
	g1 = premio * 0.46;
	g2 = premio * 0.32;
	g3 = premio - g1 - g2;
	
	printf("O primeiro ganhador recebera R$%.2f\nO Segundo recebera R$%.2f\nO Terceiro recebera R$%.2f\n", g1, g2, g3);
	
	return 0;
}

File: 1_Periodo/Lista 5/Python/L5_Q6.py
premio = float(780000)

ganhador1 = premio * 0.46
ganhador2 = premio * 0.32
ganhador3 = premio - ganhador2 - ganhador1

ganhador1 = f'R${ganhador1:_.2f}'
ganhador1 = ganhador1.replace('.',',').replace('_','.')

print(f"Primeiro ganhador: {ganhador1}")

ganhador2 = f'R${ganhador2:_.2f}'
ganhador2 = ganhador2.replace('.',',').replace('_','.')

print(f"Segundo ganhador: {ganhador2}")

ganhador3 = f'R${ganhador3:_.2f}'
ganhador3 = ganhador3.replace('.',',').replace('_','.')

print(f"Terceiro ganhador: {ganhador3}")